tests/acceptance/features/apiSpaces/lockSpaces.feature

```diff
@@ -0,0 +1,93 @@
+@api @skipOnOcV10
+Feature: lock
+  # Note: This Feature includes all the tests from core (apiWebdavLock suite) related to /Shares since in core no implementation is there for space Shares Jail
+
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+      | Carol    |
+    And user "Alice" has created folder "PARENT"
+    And user "Brian" has created folder "PARENT"
+    And user "Carol" has created folder "PARENT"
```

Comment:
Why are we creating these folders even though none of the scenarios are using them?

Reply:
i have a branch rebased from this branch which uses this folders so i have added it to this PR.